### PR TITLE
Slider and all sub-components: TS-migration

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/2861766393/pulses/3394912284

### CODE REVIEW

File: src/components/Slider/SliderBase/SliderBase.tsx [hunk 1]
```diff
@@ -54,19 +56,19 @@ const SliderBase = forwardRef(({ className }, ref) => {
...
     }
     if (isPageDownEvent(e)) {
       e.preventDefault();
-      return decreaseValue(calculatePageStep({ min, max, step }));
+      return decreaseValue(calculatePageStep(max, min, step));
     }
     if (isHomeEvent(e)) {
       e.preventDefault();
-      return changeThumbValue(max);
+      return changeThumbValue(min);
```
Comment (niksa-monday) [inline src/components/Slider/SliderBase/SliderBase.tsx:67]: Did you intentionally change max to min or is it a mistake?

File: src/components/Slider/SliderBase/SliderBase.tsx [hunk 2]
```diff
@@ -54,19 +56,19 @@ const SliderBase = forwardRef(({ className }, ref) => {
...
     }
     if (isEndEvent(e)) {
       e.preventDefault();
-      return changeThumbValue(min);
+      return changeThumbValue(max);
```
Comment (niksa-monday) [inline src/components/Slider/SliderBase/SliderBase.tsx:71]: Same here: Did you intentionally change min to max or is it a mistake?

File: src/components/Slider/SliderBase/SliderBase.tsx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Slider/SliderBase/SliderBase.tsx:67]: Intentionally made this and forgot to mention - I think it was bugged before. Home button should make it to 0 and end button should make it to 100 - that's logical, but it was the other way around before.
It's even more obvious on Mac, fn + arrow left made it to 100 and fn + arrow right made it to 0 - not correct

Comment (niksa-monday) [reply src/components/Slider/SliderBase/SliderBase.tsx:67]: Agree! 👍

**Review by niksa-monday — APPROVED** [review]

**Pull request merged**